docs/public/_redirects

```diff
@@ -476,7 +476,7 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /base/api/use-tabs/ /base-ui/react-tabs/hooks-api/#use-tabs 301
 /base/api/use-tabs-list/ /base-ui/react-tabs/hooks-api/#use-tabs-list 301
 /base/* /base-ui/:splat 301
-/base-ui/api/* /base-ui/react-:splat/components-api/#:splat
+/base-ui/api/:component/ /base-ui/react-:component/components-api/#:component 301
```

Comment:
`curl -I https://mui.com/base-ui/api/portal/` to see why, it's a double 301. Also, the leading slash breaks the hash anchor link, and finally, we missed the 301.

Reply:
`curl -I https://deploy-preview-37594--material-ui.netlify.app/base-ui/api/portal/` is correct now